### DISCUSSION TITLE
[Auth] [Android] Option to increase size of TOTPs

### DISCUSSION THREAD

**Discussion by maxstarkenburg** [ente-io/ente#4120] [Enhancements]
Yes, I need to get new glasses, but in the meanwhile ...

I would love for the one-time passwords to be larger than they currently are. I appreciate that on some other authentication apps, the numbers display bold and in a much bigger font.

I wouldn't mind if this were an optional setting I could change (if not just a default change).

**Comment by dnred** [comment]
Also on iOS especially in compact mode.

Also #3557
